Src/DependencyCollector/Shared/HttpDependenciesParsingTelemetryInitializer.cs

```diff
@@ -96,5 +98,49 @@ private bool TryParseAzureBlob(string host, string name, string url, out string
 
             return result;
         }
+
+        private bool TryParseAzureTable(string host, string name, string url, out string account, out string verb, out string tableName)
+        {
+            bool result = false;
+
+            account = null;
+            verb = null;
+            tableName = null;
+
+            if (name != null && host != null && url != null)
+            {
+                if (host.EndsWith("table.core.windows.net", StringComparison.OrdinalIgnoreCase))
```

Comment:
Would it work and be safer to add a period in front of the domain ".table.core..."?